README.md

```diff
@@ -1,2 +1,104 @@
 # re-route
-Routing wrapper library for SPAs based on re-frame.
+
+Routing wrapper library for SPAs based on [re-frame](https://github.com/day8/re-frame) and [reitit](https://github.com/metosin/reitit).
+
+In addition to standard navigation functionality, this library comes with built-in functionality from navigation prevention, e.g. to warn the user of unsaved input changes.
```

Comment:
"built-in functionality *for* navigation prevention"

Reply:
https://github.com/yetanalytics/re-route/pull/1/commits/e3c9c74ea6f1e6ff232c19f73953b0fea1de7296